landing/templates/landing/team.html

```diff
@@ -0,0 +1,163 @@
+{% extends "firstbyte/base.html" %}
+{% load static %}
+{% block title %}Meet the Team{% endblock %}
+{% block head %}
+    <link href="https://fonts.googleapis.com/css?family=Tinos:400i" rel="stylesheet">
+{% endblock %}
+{% block meta %}
+{% endblock %}
+{% block content %}
+<section>
+    <div class="container-full-bg" style="background-image: linear-gradient( rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.5) ), url('{% static 'landing/img/snell-team.jpg' %}')">
+        <div class="container special text-center text-white">
+            <div class="py-4">
+                <table class="pt-5" style="height:30vw; width: 100%;">
+                    <tbody><tr>
+                        <td class="align-bottom">
+                            <h1 class="font-weight-bold">The FirstByte Team</h1>
+                            <p class="lead">Get your first taste of teaching tech.</p>
```

Comment:
I feel like we should move `We are a team of undergraduate students at Northeastern University.` to this subtitle position and put `FirstByte is composed of 5 groups that work together as one team:` where that used to be--
@cyan478 can finalize the copy.